tb_plugin/README.md

```diff
@@ -258,11 +272,76 @@ The suspended toolbar has functionalities to help view the trace line.
 
 The “Optimizer.step#SGD.step” and ”enumerate(DataLoader)#_SingleProcessDataLoaderIter.\__next\__”
 are high-level python side functions.
 
 When you select the top-right corner's “Flow events” to ”async”,
 you can see the relationship between an operator and its launched kernels.
-![Alt text](https://github.com/pytorch/kineto/blob/master/tb_plugin/docs/images/trace_view_launch.PNG)
+![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/trace_view_launch.PNG)
+
+You can also view the gpu utilization and Est. SM Efficiency in the trace view. They are drawn alongside the timeline:
+
+![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/trace_view_gpu_utilization.PNG)
+
+* Memory View
+
+    Pytorch profiler records all memory allocation/release events during profiling. For each operator, the plugin aggregates all the events
+    inside its life span.
+
+    ![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/memory_view.PNG)
+
+    The memory kind could be selected in “Device” selection box. For example, “GPU0” means the following table only shows each operator’s memory usage on GPU 0, not including CPU or other GPUs. 
+
+    Definition of each field in the table:
+
+
+    * Calls: How many times this operator is called. 
+
+    * Size Increase: The memory increase size include all children operators. It sums up all allocation bytes and minus all the memory release bytes. 
+
+    * Self Size Increase: The memory increase size associated with the operator itself excluding that of its children. It sums up all allocation bytes and minus all the memory release bytes. 
+
+    * Allocation Count: The allocation count including all children operators. 
+
+    * Self Allocation Count: The allocation count belonging to the operator itself excluding its chilren. 
+
+    * Allocation Size: The allocation size including all children operators. It sums up all allocation bytes without considering the memory free. 
+
+    * Self Allocation Size: The allocation size belonging to the operator itself. It sums up all allocation bytes without considering the memory free.
+
+
+* Distributed View
+
+    This view will appear automatically only for DDP jobs that use nccl for communication.
+    There are four panes in this view: 
+
+    ![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/distributed_view.PNG)
+
+    *   The top pane shows the information about nodes/processes/GPU hierarchy of this job.
+    
+    *   The left pane in the middle is 'Computation/Communication Overview'. Definition of each legend:
+        * Computation: the sum of kernel time on GPU minus the overlapping time
+        * Overlapping: the overlapping time of computation and communication. More overlapping represents better parallelism between computation and communication. Ideally the computation and communication totally overlap with each other. 
```

Comment:
Last sentence: "Ideally the communication could be totally overlapped with computation."

Reply:
done, thanks